scripts/r.in.wms/wms_gdal_drv.py

```diff
@@ -127,6 +131,11 @@ def _download(self):
         temp_map = self._tempfile()
 
         xml_file = self._createXML()
+
+        if hasattr(self, 'proxy'):
```

Comment:
I would prefer setting the proxy and proxy_auth to None in the beginning, rather than using hasattr, that's more used when you ask about interface. Not a big issue, just a matter of style.

Reply:
+1

Reply:
Okay I tried it with the super. I hope this is the correct way?